server/test/repository/ApplicationRepositoryTest.java

```diff
@@ -144,8 +145,11 @@ private Applicant saveApplicant(String name) {
     return applicant;
   }
 
-  private Program saveProgram(String name) {
-    Program program = new Program(name, "desc", name, "desc", "", DisplayMode.PUBLIC.getValue());
+  private Program createProgram(String name) {
+    Version version = new Version();
+    version.save();
```

Comment:
I think we have to find the current draft, if we save two draft versions this way that's a system error that likely will fail other code.

I'm kinda surprised it's not a problem in this class but the code may not be exercising flows that require the singleton Draft version.